packages/ui/src/views/chatmessage/ChatExpandDialog.js

```diff
@@ -43,7 +43,7 @@ const ChatExpandDialog = ({ show, dialogProps, onClear, onCancel }) => {
                     )}
                 </div>
             </DialogTitle>
-            <DialogContent>
+            <DialogContent sx={{ display: 'flex', justifyContent: 'flex-end', flexDirection: 'column' }}>
```

Comment:
tried this and didnt see any changes on ui, whats this for?

Reply:
This happen when i paste text into input without this code

https://github.com/FlowiseAI/Flowise/assets/124426279/c504a2a1-de6c-4cd7-b37a-3bffe511b5b4

Reply:
I can fix  this.